perceval/backends/core/jenkins.py

```diff
@@ -265,6 +276,8 @@ def setup_cmd_parser():
         group.add_argument('--sleep-time', dest='sleep_time',
                            type=int, default=SLEEP_TIME,
                            help="Minimun time to wait after a Timeout connection error.")
+        parser.parser.add_argument('--username', dest='username', type=str, default=None, help="Jenkins HTTP username")
```

Comment:
username and password should go to group, since they are optional params

Reply:
Why don't you activate `basic_auth`? https://github.com/chaoss/grimoirelab-perceval/blob/master/perceval/backend.py#L273

Reply:
good point @sduenas, better to use basic_auth